sql: TestTrace failed

sql.TestTrace [failed](https://tanzanite.cluster.engflow.com/invocation/340c9760-de79-476a-a36e-c876a70d951b) on master @ [64dacab2808c5420e3fd99d2562e8ecb97b7742b](https://github.com/cockroachdb/cockroach/commits/64dacab2808c5420e3fd99d2562e8ecb97b7742b):


```
=== RUN   TestTrace
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestTrace3615755377
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:146: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
    trace_test.go:421: -- test log scope end --
test logs left over in: outputs.zip/logTestTrace3615755377
--- FAIL: TestTrace (3.53s)
=== RUN   TestTrace/Session
    --- FAIL: TestTrace/Session (0.16s)
=== RUN   TestTrace/Session/TracingOn
        --- FAIL: TestTrace/Session/TracingOn (0.07s)
=== RUN   TestTrace/Session/TracingOn/node-2
    trace_test.go:395: expected span: "commit sql txn", got: "admissionWorkQueueWait"
    trace_test.go:406: remaining span: "commit sql txn"
    trace_test.go:406: remaining span: "consuming rows"
    trace_test.go:406: remaining span: "dist sender send"
    trace_test.go:406: remaining span: "flow"
    trace_test.go:406: remaining span: "optimizer"
    trace_test.go:406: remaining span: "session recording"
    trace_test.go:406: remaining span: "sql query"
    trace_test.go:406: remaining span: "sql txn"
    trace_test.go:406: remaining span: "table reader"
    trace_test.go:406: remaining span: "txn coordinator send"
            --- FAIL: TestTrace/Session/TracingOn/node-2 (0.02s)
```
<p>Parameters: <code>attempt=1</code>
, <code>run=11</code>
, <code>shard=8</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTrace.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-33682

--- a/pkg/sql/trace_test.go
+++ b/pkg/sql/trace_test.go
@@ -51,6 +51,7 @@ func TestTrace(t *testing.T) {
 		"request range lease",
 		"range lookup",
 		"local proposal",
+		"admissionWorkQueueWait",
 	}
 
 	testData := []struct {

